Source/WebCore/rendering/style/GridPositionsResolver.cpp

```diff
@@ -181,13 +181,18 @@ void NamedLineCollectionBase::ensureInheritedNamedIndices()
 
 bool NamedLineCollectionBase::contains(unsigned line) const
 {
+    ASSERT(hasNamedLines());
```

Comment:
I feel like a `contains` method should have an assert like this, but I kept it in for now since it was in the subclasses implementations